File: blog/src/main/java/elixter/blog/domain/Image.java
package elixter.blog.domain;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class Image {
    private Long id;
    private String originName;
    private String url;
    private String status;

    public Image() {
    }

    public Image(String originName, String url, String status) {
        this.originName = originName;
        this.url = url;
        this.status = status;
    }

    public Image(Long id, String originName, String url, String status) {
        this.id = id;
        this.originName = originName;
        this.url = url;
        this.status = status;
    }
}

File: ImageRemover/model/image.go
package model

import "time"

type Image struct {
	Id         int64     `db:"id"`
	OriginName string    `db:"origin_name"`
	Url        string    `db:"url"`
	CreateAt   time.Time `db:"create_at"`
	Status     string    `db:"status"`
}
